src/calculateRentalCost.js

```diff
@@ -3,8 +3,20 @@
  *
  * @return {number}
  */
-function calculateRentalCost(days) {
-  // write code here
-}
+function calculateRentalCost(days) { 
+  const PRICE_PER_DAY = 40; 
+  const LONG_TERM = 7;
+   const LONG_TERM_DISCOUNT = 50; 
+   const SHORT_TERM = 3; 
+   const SHORT_TERM_DISCOUNT = 20;
 
+    let basePrice = days * PRICE_PER_DAY;
+     if (days >= LONG_TERM) { 
+      return basePrice - LONG_TERM_DISCOUNT;
+     } 
+      if (days >= SHORT_TERM) {
+         return basePrice - SHORT_TERM_DISCOUNT;
```

Comment:
Consider simplifying the logic by using `if` with `return` to avoid using `else` after `if` with `return`. This will make the code more readable.